css/src/components/progress-bar.scss

```diff
@@ -0,0 +1,99 @@
+$progress-bar-height-medium: 2px;
+$progress-bar-height-large: 4px;
+
+$progress-bar-background: $secondary-background !default;
+$progress-bar-color: $primary !default;
+$progress-bar-success-color: $success !default;
+$progress-bar-warning-color: $warning !default;
+$progress-bar-danger-color: $danger !default;
+$progress-bar-border-radius-rounded: $border-radius-rounded !default;
+
+$progress-bar-animation-duration: 2s;
+
+.progress-bar-container {
+	position: relative;
+	width: 100%;
+	border-radius: $progress-bar-border-radius-rounded;
+	background-color: $progress-bar-background;
+	overflow: hidden;
+
+	.theme-high-contrast & {
+		background-color: $secondary-background-glow-high-contrast;
+	}
+
+	&.progress-bar-md {
+		height: $progress-bar-height-medium;
+	}
+
+	&.progress-bar-lg {
+		height: $progress-bar-height-large;
+	}
+
+	&.progress-bar-square {
```

Comment:
That should be default, no atomic required

Reply:
@ellynliu Per design, corners are rounded. Please make it default 🙏🏻

![image](https://github.com/user-attachments/assets/40d125bc-a761-4cdd-8ad2-f99503780d1b)

Reply:
pushed

Reply:
Remove that completely. In case someone will need to reset the border radius they'll use the border atomic

Reply:
got it, removed completely